server/src/chatbot.py

```diff
@@ -32,21 +38,11 @@ def preprocess_input(user_message):
     processed_message = ' '.join(tokens)
     return processed_message
 
-def match_intent_to_output(intent_list):
-    # Loop through each word in the processed message
-    output = []
-    for word in intent_list.split():
-        # Match the word to an intent
-        # For now, we'll just return a placeholder intent
-        if word == 'hello':
-            output.append('greetings')
-    
-    if len(output) == 0:
-        output.append('no_intent')
-    return output
-        
-def match_input_to_intent(processed_message):
-    # Placeholder
-    return processed_message
+def match_intent_to_response(intent, intent_response):
+    if intent in intent_response:
+        return intent_response[intent]
+    else: 
+        return "I'm sorry, I'm not sure if I understand. Could you provide me with more details please?"
```

Comment:
Say, that the conversation goes like this.

```
"Is the wifi down? I can't seem to connect"

"I'm sorry, I'm not sure if I understand. Could you provide me with more details please?"

"Well I tried reconnecting several times, but it doesn't seem to be working."
```

We don't have the functionality to recognize that something is a detail to an inquiry rather than a new inquiry. I think we need to change the "... more details please?" to "Try again" or something along those lines.